pkgs/by-name/de/dep-tree/package.nix

```
@@ -0,0 +1,62 @@
+{lib, fetchFromGitHub, buildGoModule, linkFarm, ... }: let
+  testDeps = {
+    react-stl-viewer = fetchFromGitHub {
+      owner = "gabotechs";
+      repo = "react-stl-viewer";
+      rev = "2.2.4";
+      sha256 = "sha256-0u9q0UgOn43PE1Y6BUhl1l6RnVjpPraFqZWB+HhQ0s8=";
+    };
+    react-gcode-viewer = fetchFromGitHub {
+      owner = "gabotechs";
+      repo = "react-gcode-viewer";
+      rev = "2.2.4";
+      sha256 = "sha256-FHBICLdy0k4j3pPKStg+nkIktMpKS1ADa4m1vYHJ+AQ=";
+    };
+    graphql-js = fetchFromGitHub {
+      owner = "graphql";
+      repo = "graphql-js";
+      rev = "v17.0.0-alpha.2";
+      sha256 = "sha256-y55SNiMivL7bRsjLEIpsKKyaluI4sXhREpiB6A5jfDU=";
+    };
+    warp = fetchFromGitHub {
+      owner = "seanmonstar";
+      repo = "warp";
+      rev = "v0.3.3";
+      sha256 = "sha256-76ib8KMjTS2iUOwkQYCsoeL3GwBaA/MRQU2eGjJEpOo=";
+    };
+  };
+in
+buildGoModule (let
+  testFarm = linkFarm "dep-tree_testDeps-farm"
+    (builtins.attrValues
+      (builtins.mapAttrs (name: value: { name = name; path = value; }) testDeps));
```

This is a bit of unnecessary eval work. `linkFarm` happens to prefer attrsets, but can take lists and convert them back to attrsets:

https://github.com/NixOS/nixpkgs/blob/db4e8b299eb00afa2ed398f90a9a317431bf1af1/pkgs/build-support/trivial-builders/default.nix#L522-L528

To avoid converting back and forth, we should just give it the attrset.

```suggestion
  testFarm = linkFarm "dep-tree_testDeps-farm" testDeps;
```

---

oooo nice.

Ok. So what I take from these 2 reviews is the following. This linkFarm thing, and get rid of the rec, and fix the spacing. Will ammend them into the commit with with the package